app/assets/stylesheets/categories.css

```diff
@@ -0,0 +1,107 @@
+#categories {
+  padding: 10px 15px;
+  margin: 10px 2%;
+  border: 1px solid #434b54;
+  border-radius: 5px;
+}
+
+.description-container {
+  display: flex;
```

Comment:
- [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```

Reply:
Thank you for this. I've always been using it before this project, but I was really focusing on getting it to work and forgot to refactor I'll hopefully do that during the weekend and I'll make the burger menu functional too. Thanks a lot @Lameck1